lib/epiviewpoint/bulk_fhir_parser.ex

```diff
@@ -139,31 +143,60 @@ defmodule EpiViewpoint.BulkFhirParser do
       testname: testname,
       result: result,
       org_id: org_id,
-      datereportedtolhd: find_extension(extensions)
+      datereportedtolhd: find_extension(extensions, :datereportedtolhd)
     }
   end
 
-  defp extract_organization(%EpiViewpoint.R4.Organization{
-         id: organization_id,
-         name: ordering_facility_name
-       }) do
+  defp extract_organization(%EpiViewpoint.R4.Organization{} = organization) do
+    %EpiViewpoint.R4.Organization{
+      id: organization_id,
+      name: ordering_facility_name
+    } = organization
+
     %{
       organization_id: organization_id,
```

Comment:
I don't think you need the pattern match here. You can just use `organization.id` and `organization.name` below.

Reply:
totally!